### ISSUE TITLE
Tear break-up time brief

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9887]
**Preferred term label:**
Tear break-up time brief

**Synonyms**
Tear film break-up time brief

**Definition (free text, please give PubMed ID)**
Tear film break up time less than 10 seconds

**Parent term (use hpo.jax.org/app)**
UMLS has C0429495 for the test, but not its result 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dry eye syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Please note I replaced 10 seconds as below, because some sources state 5 seconds.

Adding new term: 
- ID: HP:6000069
- label: Brief tear break-up time
- parents: [Lacrimation abnormality (HP:0000632)](https://hpo.jax.org/app/browse/term/HP:0000632)
- pmid: PMID:30481808
- def: Tear film break up time (number of seconds that elapse between the last blink and the appearance of the first dry spot in the tear film) below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9887
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Please note I replaced 10 seconds as below, because some sources state 5 seconds.

Adding new term: 
- ID: HP:6000069
- label: Brief tear break-up time
- parents: [Lacrimation abnormality (HP:0000632)](https://hpo.jax.org/app/browse/term/HP:0000632)
- pmid: PMID:30481808
- def: Tear film break up time (number of seconds that elapse between the last blink and the appearance of the first dry spot in the tear film) below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9887
- synonyms:
